pages/treeherder.py

```diff
@@ -64,3 +73,22 @@ def __init__(self, base_url, selenium):
     @property
     def is_job_status_visible(self):
         return self.is_element_visible(self._job_header_locator)
+
+
+class SingleResultPage(Page):
```

Comment:
It's true that the format of the page does not change, but the url does- which is why a page region made sense to me. You cannot view the single result without an action. You could also say the same about the logviewer - it is part of the treeherder page but it only displays with an action. It's confusing to me.

Lastly I totally agree about the locators.